### PR TITLE
DRIVERS-1954: Minor fixes

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

**Review by nbbeeken — APPROVED** [review]
LGTM, thanks for the fixes!
@ShaneHarvey I figured we don't need a new ticket for these changes since we know everyone's blocked on the other work, so no one should fall out of sync. I can update the Downstream Changes Summary field to call this out after we merge.

**Review by ShaneHarvey — APPROVED** [review]
SGTM.

**Pull request merged**
